app/models/base_facility.rb

```diff
@@ -1,72 +1,15 @@
 # frozen_string_literal: true
 
-require 'facilities/shared_client'
+require 'facilities/client'
 
 class BaseFacility < ActiveRecord::Base
+  include Facilities::FacilityMapping
```

Comment:
I'm seeing Facilities::FacilityMapping get mixed in everywhere, but it's no longer clear to me whether or not all of these constants are required for everywhere it is being mixed in. Should we maybe consider breaking it down more granularly into separate mixins and if certain components only exist 1 class, not including it as a mixin at all?

I see the benefit here, but it also makes reading the code more difficult because you'll see a constant defined and its not clear immedietely where it is from. Perhaps instead of mixing in the module, reference the constants using the namespace:

Facilities::FacilityMapping::PathMap or something like that which might be better for readability (NITPICK)